SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverRPC.cpp

```diff
@@ -381,11 +381,19 @@ bool FSpatialNetDriverRPC::ApplyRPC(Worker_EntityId EntityId, const FRPCPayload&
 	if (TargetObject == nullptr)
 	{
 		const TWeakObjectPtr<UObject> ActorReceivingRPC = NetDriver.PackageMap->GetObjectFromEntityId(EntityId);
-		AActor* Actor = CastChecked<AActor>(ActorReceivingRPC.Get());
-		checkf(Actor != nullptr, TEXT("Receiving actor should have been checked in CanReceiveRPC"));
-		UE_LOG(LogSpatialNetDriverRPC, Error,
-			   TEXT("Failed to execute RPC on Actor %s (Entity %llu)'s Subobject %i because the Subobject is null"), *Actor->GetName(),
-			   EntityId, RPCData.Offset);
+		if (UObject* ActorReceiving = ActorReceivingRPC.Get())
+		{
+			AActor* Actor = CastChecked<AActor>(ActorReceiving);
+			checkf(Actor != nullptr, TEXT("Receiving actor should have been checked in CanExtractRPC"));
```

Comment:
Nit : The diagnosis in checkf should be changed to something like "Unexpected RPC recipient, should be an actor"